sktime/forecasting/compose/_fallback.py

```diff
@@ -209,7 +235,7 @@ def _predict(self, fh, X=None):
             return self.current_forecaster_.predict(fh, X)
         except Exception as e:
             self.exceptions_raised_[self.first_nonfailing_forecaster_index_] = {
-                "failed_at_step": "predict",
+                "failed_at_step": "fit",
```

Comment:
I think you are accidentally reverting @yarnabrina's fix, you need to update your branch from `main`

Reply:
I suspect this happened due to rebase on an earlier state - to prevent this, imo, merge `main` into branch (rather than rebase) is safer

Reply:
Oops, good catch